src/components/icon/assets/polygon.svg

```diff
@@ -4,5 +4,6 @@
   -->
 
 <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
-  <path d="M1 5V1H5V2.29999H11V1H15V5H12.0658L11.2581 6H12V10H11.2403L12.0403 11H15V15H11V13.5H5V15H1V11H2.5V5H1ZM2 2H4V4H2V2ZM11 12.5V11.3004L9.95972 10H8V6H9.97266L11 4.72806V3.29999H5V5H3.5V11H5V12.5H11ZM12 2H14V4H12V2ZM4 12H2V14H4V12ZM9 7H11V9H9V7ZM12 12H14V14H12V12Z"/>
-</svg>
+  <path
+    d="M15 11v4h-4v-1.5H5V15H1v-4h1.5V5H1V1h4v1.3h6V1h4v4h-2.9l-.8 1h.7v4h-.8l.8 1h3zm-1 3v-2h-2v2h2zm0-10V2h-2v2h2zm-3 8.5v-1.2L10 10H8V6h2l1-1.3V3.3H5V5H3.5v6H5v1.5h6zM11 9V7H9v2h2zm-7 5v-2H2v2h2zM4 4V2H2v2h2z" />
+</svg>
```

Comment:
Can we get a blank line at the end of the file?

Reply:
No problem.